Test/UnitTest1.cs

```diff
@@ -0,0 +1,86 @@
+using Training;
+
+namespace MSTestProject {
+   [TestClass]
+   public class UnitTest1 {
+      [TestMethod]
+      public void AddTest () {
+         myList.Add (1);
+         myList.Add (2);
+         list.Add (1);
+         list.Add (2);
+         Assert.AreEqual (list.Count, myList.Count);
+         Assert.AreEqual (list.Capacity, myList.Capacity);
+         myList.Clear ();
+         list.Clear ();
+      }
+
+      [TestMethod]
+      public void RemoveTest () {
+         myList.Add (1);
+         myList.Add (2);
+         list.Add (1);
+         list.Add (2);
+         myList.Remove (1);
+         list.Remove (1);
+         Assert.AreEqual (list.Count, myList.Count);
+         myList.Clear ();
+         list.Clear ();

```

Comment:
The Remove (..) method here will return a boolean and the tests should cover both (i.e) when the method returns "true" or "false".